docs/resources/google_cloudbuild_trigger.md

```diff
@@ -87,12 +89,52 @@ Properties that can be accessed from the `google_cloudbuild_trigger` resource:
 
   * `build`: Contents of the build template. Either a filename or build template must be provided.
 
+    * `source`: The location of the source files to build.
+
+      * `storage_source`: Location of the source in an archive file in Google Cloud Storage.
+
+        * `bucket`: Google Cloud Storage bucket containing the source.
+
+        * `object`: Google Cloud Storage object containing the source. This object must be a gzipped archive file (.tar.gz) containing source to build.
+
+        * `generation`: Google Cloud Storage generation for the object.  If the generation is omitted, the latest generation will be used
```

Comment:
There's a bunch of instances of double-spacing within this file, this would probably need to be updated upstream though right?

Reply:
Suggest we proceed with what's here but you're right, the spacing could be tweaked in a future MM PR to ensure consistency.